eng/common/templates/phases/base.yml

```diff
@@ -0,0 +1,107 @@
+parameters:
```

Comment:
I presume all the stuff under `eng/common` is just a straight port from Arcade, I don't intend to review it.

Reply:
Correct.